pulp_service/pulp_service/app/models.py

```diff
@@ -18,7 +18,7 @@
 from django.contrib.postgres.fields import ArrayField
 
 from pulpcore.plugin.models import Domain
-from pulpcore.app.models import AutoAddObjPermsMixin, HeaderContentGuard
+from pulpcore.app.models import AutoAddObjPermsMixin, HeaderContentGuard, BaseModel
```

Comment:
You didn't create this problem, but can you try switching the imports from `pulpcore.app` to `pulpcore.plugin`. Over-time directly importing from pulpcore other than `pulpcore.plugin` isn't safe from breaking changes. The "plugin API" though is safe which is offered inside `pulpcore.plugin`. Let me know if you have quesitons, or if you find you don't have all the objects you need.

Reply:
Actually this is all-over our codebase. Let's do this as a separate activity not in this PR.

Reply:
https://issues.redhat.com/browse/PULP-391